index.bs

```diff
@@ -3763,10 +3772,653 @@ nothrow>WritableStreamDefaultControllerError ( <var>controller</var>, <var>error
 
 <h2 id="ts">Transform Streams</h2>
 
-Transform streams have been developed in the testable implementation, but not yet re-encoded in spec language.
-We are waiting to validate their design before doing so. In the meantime, see <a
-href="https://github.com/whatwg/streams/blob/master/reference-implementation/lib/transform-stream.js">reference-implementation/lib/transform-stream.js</a>.
+<h3 id="ts-intro">Using Transform Streams</h3>
+
+<div class="example" id="example-basic-pipe-through">
+  The natural way to use a transform stream is place it in a <a lt="piping">pipe</a> between a <a>readable stream</a>
```

Comment:
place -> placing?

Reply:
We could use "by placing it" or "to place it". I think I prefer "to place it".